doc/recognition.xml

```diff
@@ -249,6 +249,15 @@ kernel have been created or recognised yet. You can use for example
 the methods in <Ref Var="FindHomMethodsPerm"/> or
 <Ref Var="FindHomMethodsMatrix"/> or <Ref Var="FindHomMethodsProjective"/>
 as the <A>method</A> argument.
+
+<P/>GAP homomorphisms are not required to give a sensible answer when
+given a value not in their source, and in practice often enter the break
+loop, or return an incorrect answer. This causes problems when checking
+if a value is not in the represented group. To avoid this problem,
+<Ref Attr="validatehomominput"/> can be set to a function.
+This function is used to filter possible group elements,
+which are not known to be in the recognised group, before they are
```

Comment:
I am not quite sure I understand this sentence correctly. I think I'd prefer if it said a bit less:
```suggestion
before they are
```